### DISCUSSION TITLE
Remaining language ports

### DISCUSSION THREAD

**Discussion by laserpants** [sqids/sqids-spec#10] [General]
Hi @4kimov, and everyone else here! :wave: I am curious to know what the status is for the remaining language implementations. I'd be interested to work on the C++(11) version, and possibly a few other languages. Any work in progress here already?

**Comment by 4kimov** [comment]
Hi @laserpants, for the remaining langs, it's either a) some people have volunteered, but haven't gotten to it yet or b) we have nobody (and I ran out of the languages that I do know). For C++ version, I don't see anybody in the maintainers list, so I've added you.

Thanks for pinging, and for anybody else reading this - please ping if you'd like to take a stab at any other lib.
